server/sockets-interface.js

```diff
@@ -95,6 +95,14 @@ module.exports = function socketsInterface(wss) {
       createTicket(val) {
         const { title } = val;
         grooming.createTicket(title);
+      },
+      startTyping(val) {
+        const { name } = val;
+        grooming.startTyping(name);
```

Comment:
Related to the comment about the server knowing the user below: instead of `name` we can use `this.username` and not require any arguments.

Reply:
Sorry, not `this.username` on the server, but just `user` (it's the variable in this closure)